mix.exs

```diff
@@ -43,6 +45,7 @@ defmodule Floki.Mixfile do
 
     [
       {:html_entities, "~> 0.5.0"},
+      {:mix_gleam, "~> 0.1.0"},
```

Comment:
Testing this as a solution to always compile Gleam source code, though the best solution is probably to compile gleam code conditionally.